tests/integration/test_fim/test_files/test_follow_symbolic_link/test_audit_rules_removed_after_change_link.py

```diff
@@ -0,0 +1,107 @@
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+import os
+from time import sleep
+import subprocess
+
+import pytest
+
+pytestmark = [pytest.mark.linux, pytest.mark.tier(level=1)]
```

Comment:
flake8 reports some style errors, specifically some imports that are made after this line (move it down) and some unused imports, please review and fix them
![imagen](https://user-images.githubusercontent.com/20206636/103908654-ffe64e80-5102-11eb-9b29-ee9de6b664cd.png)

Reply:
The only warning that appears now is the unused import of the variable `test_directories`.  This variable is used in the conftest in order to create the directories where the tests are executed. The rest of the warnings are solved.